docs/release-notes.md

```diff
@@ -1,45 +1,41 @@
-
-#### [Sample UI](Glossary.md#s)
-- **New look & feel** The Sample UI has had a full rewrite in the 1.2 release based on much feedback from our customers.  This updated UI supports the new hostname and metadata tag features.  While this is still just a reference implementation to showcase the features of the MPS and RPS components, this restructuring of the Sample UI will:
-    * Aid developers as an example of how to use our APIs 
-    * Help understand the value of Open AMT Cloud Toolkit.
 
-### Modifications and Removals
-#### [RPS](Glossary.md#r)
-- **RPS API changes** We have performed some fundamental restructuring of the RPS REST APIs to better align with industry RESTful API standards.  With these changes you'll see a greatly simplified structure, error messages in proper JSON structure, consistent property names across request/response, and correct HTML status codes being returned.  *This is a breaking change for customers who have already implemented against the 1.1 API.*
-#### [UI Toolkit](Glossary.md#u)
-- **Deprecation announcement of non-socket UI Toolkit components:** We are removing the following UI Toolkit components in the 1.3 release: Audit Log, CIRA Config, Device Grid, Domain, and Profile.  All of these components are just UI implementations of our existing REST APIs and the need for a REACT component for these items is unnecessary.  We will continue to support and maintain the REACT components for KVM and SOL.  Additionally, we have plans on expanding the KVM and SOL components to support additional UI frameworks beyond REACT.
-#### [MPS](Glossary.md#m) & [RPS](Glossary.md#r)
-- **Deprecation announcement of Developer Mode for both RPS and MPS** We are making strides in improving production mode deployments through the use of deployment scripts and docker containers.  Due to this work, the need for a "simple" developer deployment mode that doesn't use Vault or PostgreSQL is no longer required.  As such, we'll be removing this mode prior to our 1.4 release.  After the removal of developer mode, Vault and PostgreSQL will be required by default but the docker deployment scripts we have will make getting these services up and running easy.  Customers will still have the option to modify the Vault and PostgreSQL interfaces to point to the database service(s) of their choice.
 
 ## Resolved Issues
+- **KVM freeze intermittently:** We have added a small delay in handling mouse interactions that prevents us from flooding the AMT channel.  There are still a few occasions where KVM could still freeze.  We are still root causing this new issue
```

Comment:
Don't feel like this can go here if we are still root causing

Reply:
Back to the known issue list it goes!